modules/secretsmanager/main.tf

```diff
@@ -1,8 +1,32 @@
 ## Local Variables
 locals {
   tags = merge(var.tags, { Deployment = var.prefix })
+  iam_users    = data.aws_iam_users.orca_users.arns
+  account_id   = data.aws_caller_identity.current_account.account_id
+  region       = data.aws_region.current_region.name
+  kms_arn    = "arn:aws:kms:${local.region}:${local.account_id}:key/*"
+  lambda_rolename = "${var.prefix}_restore_object_role"
 }
 
+
+## Data Lookups
```

Comment:
I'd be tempted to move these to the top of the file, just to make the section that uses them a tad clearer. Know it wouldn't change anything functionally.